bootstrap/opnsense-bootstrap.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-# Copyright (c) 2015-2020 Franco Fichtner <franco@opnsense.org>
+# Copyright (c) 2015-2021 Franco Fichtner <franco@opnsense.org>
```

Comment:
Very kind but since there we no substantial changes this year it can probably be deferred :)